packages/web-components/src/components/notice-choice/notice-choice.ts

```diff
@@ -56,6 +56,9 @@ class NoticeChoice extends StableSelectorMixin(LitElement) {
   @property({ type: String, attribute: 'language' })
   language = 'en';
 
+  @property({ type: String, attribute: 'currentLanguage' })
```

Comment:
```suggestion
  @property({ type: String, attribute: 'current-language' })
```

Reply:
This is not yet resolved. Attribute names are snake case by convention, not camel case. Any particular reason to go against the convention for this one out of all the others?